packages/calling/src/CallingClient/calling/callManager.ts

```diff
@@ -421,6 +429,21 @@ export class CallManager extends Eventing<CallEventTypes> implements ICallManage
   public getActiveCalls = (): Record<string, ICall> => {
     return this.callCollection;
   };
+
+  /**
+   * @param deviceId -.
+   * @param line -.
+   */
+  public addLine(deviceId: string, line: ILine) {
```

Comment:
this should be a private function

Reply:
In register function in line class, we are calling this function.